website/docs/r/datadog_tagrules.html.markdown

```diff
@@ -0,0 +1,100 @@
+---
+subcategory: "Datadog"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_datadog_monitor_tagrules"
+description: |-
+  Manages TagRules on the datadog Monitor.
+---
+
+# azurerm_datadog_monitor_tagrules
+
+Manages TagRules on the datadog Monitor.
+
+## Example Usage
+
+### Adding TagRules on monitor
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-datadog"
+  location = "West US 2"
+}
+resource "azurerm_datadog_monitor_tagrules" "example" {
+  name                = "example-monitor"
+  resource_group_name = azurerm_resource_group.example.name
+  log_rules{
+    send_subscription_logs = true
+  }
+  metric_rules{
+    filtering_tag {
+        name = "Test"
+        value = "Logs"
+        action = "Include"
+    }
+  }
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name which should be used for this datadog Monitor.
+
+* `resource_group_name` - (Required) The name of the Resource Group where the datadog Monitor should exist.
```

Comment:
Can we use `datadog_monitor_id` instead?

Reply:
Done